src/network/clientpackethandler.cpp

```diff
@@ -1328,6 +1328,9 @@ void Client::handleCommand_HudSetStars(NetworkPacket *pkt)
 
 	*pkt >> stars.visible >> stars.count
 		>> stars.starcolor >> stars.scale;
+	try {
+		*pkt >> stars.visible_at_day;
+	} catch (PacketError &e) {};
```

Comment:
Uninitialized `visible_at_day`. This should ensure a proper fallback, also for future extensions:
`StarParams stars = SkyboxDefaults().getMoonDefaults();`